core/designsystem/src/commonMain/kotlin/io/github/droidkaigi/confsched/designsystem/component/ClickableLinkText.kt

```diff
@@ -41,7 +41,7 @@ fun ClickableLinkText(
     overflow: TextOverflow = TextOverflow.Clip,
     maxLines: Int = Int.MAX_VALUE,
     hasUnderLine: Boolean = true,
-    underLineColor: Color = LightBlue,
+    underLineAndLinkTextColor: Color = LightBlue,
```

Comment:
In practice, since it also changes the color of parts matched by regular expressions other than underlines, I changed the argument names to reflect the actual behavior.